cmd/audit/audit.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"log"
+
+	"github.com/spf13/cobra"
+)
+
+func main() {
+	rootCmd := &cobra.Command{
+		Use:   "audit",
+		Short: "Tool for auditing flyctl commands",
+	}
+
+	rootCmd.AddCommand(newListCmd())
```

Comment:
what are your thoughts on adding this as a fly command, but making it only build for dev builds? Either that, or having some kind of documentation in the README? This is one of those really interesting features of flyctl that feels like it'll get lost

Reply:
Yeah it'll get added to dev/contribution docs (when we write them 😉)